modules/mutect2/2.0/config/default.yaml

```diff
@@ -6,12 +6,18 @@ lcr-modules:
             # Available wildcards: {sample_id} {seq_type} {genome_build}
             sample_bam: "__UPDATE__"
             sample_bai: "__UPDATE__"
+            # candiate_positions may be a VCF, BED, or GATK interval list file. 
+            # For capture data, this can be used to specify the capture space. 
+            # If you don't want to provide candidate positions, leave this field as-is. 
+            # Set additional interval parameters under options: mutect2_interval_rules. 
+            # See https://gatk.broadinstitute.org/hc/en-us/articles/360035531852-Intervals-and-interval-lists for how-to. 
+            candidate_positions: "" 
 
         scratch_subdirectories: [] # Recommended: mutect2
 
         options:
-            mutect2_run_matched_unmatched: ""
```

Comment:
Why did these ones get removed/replaced with other options if the new options are still left blank?

Reply:
This is actually really a bug fix. I changed the keyword in the Snakefile to `mutect2_run` on an earlier version but didn't fix the config when I did that.